library/ssl_tls.c

```diff
@@ -4171,6 +4159,18 @@ static uint16_t ssl_preset_default_sig_algs[] = {
     MBEDTLS_TLS1_3_SIG_RSA_PSS_RSAE_SHA256,
 #endif /* MBEDTLS_X509_RSASSA_PSS_SUPPORT && MBEDTLS_SHA256_C */
 
+#if defined(MBEDTLS_RSA_C) &&  defined(MBEDTLS_SHA512_C)
```

Comment:
Doesn't this (and the whole PR) make the comment prior to this invalid?

Reply:
comment is removed